drivers/flash/soc_flash_mcux.c

```diff
@@ -89,7 +89,7 @@ static int flash_mcux_write(struct device *dev, off_t offset,
 	addr = offset + priv->config.PFlashBlockBase;
 
 	key = irq_lock();
-	rc = FLASH_Program(&priv->config, addr, (u32_t *) data, len);
+	rc = FLASH_Program(&priv->config, addr, (uint32_t *) data, len);
```

Comment:
Any particular reason `uint32_t` is used here?

Reply:
Because for some reason the KW41Z XCVR code causes the use of NewLib. NewLib uses a different typedef for u32_t that is not portable to a uint32_t which results in the gcc compiler producing a warning. I put some of the details in the check-in notes. GCC will warn that an 'unsigned int' is not generally equal to 'uint32_t' because the size of an 'unsigned int' is an architecture specific item while C ensures that a uint32_t is always 32 bits.